ruby_version_file_parser_test.go

```diff
@@ -0,0 +1,101 @@
+package bundleinstall_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	bundleinstall "github.com/paketo-buildpacks/bundle-install"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+const RUBY_VERSION_FILE_TEMPLATE = "%s\n"
+
+func testRubyVersionFileParser(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		path   string
+		parser bundleinstall.RubyVersionFileParser
+
+		versions = []string{
+			"2",
+			"2.6",
+			"2.6.0",
+			"2.7.0",
+			"2.10.0",
+			"2.10.10",
+			"10.0.0",
+		}
+	)
+
+	it.Before(func() {
+		file, err := os.CreateTemp("", ".ruby-version")
+		Expect(err).NotTo(HaveOccurred())
+		defer file.Close()
```

Comment:
I'd rewrite this to look like:

```
path = filepath.Join(t.TempDir(), ".ruby-version")
```

And then remove the cleanup from the `it.After` (see https://pkg.go.dev/testing#T.TempDir).

Reply:
> So I'd say the cleanup is not needed?

Yup, you got it